encodercontrol.cpp

```diff
@@ -74,6 +80,7 @@ void EncoderControl::saveSettings()
     qDebug() << "New position port settings saved.";
 }
 
+// QA: the same question with agrument here. Why not const QString& str? 
```

Comment:
Here, I think, you're right. I locally create QString and locally call analyzePosition(). So it shuold be working.